test/services/permissions.test.js

```diff
@@ -101,8 +90,8 @@ describe('services > permissions', () => {
                 },
               });
 
-            await new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'browseEnabled', { userId: 1 })
-              .then(() => { expect(true).toStrictEqual(true); });
+            expect(() => new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'browseEnabled', { userId: 1 }))
+              .not.toThrow();
```

Comment:
I'm pretty sure this is not the right syntax 🤔 
```suggestion
const permissionsChecker = new PermissionsChecker('envSecret', 1);
await expect(permissionsChecker.checkPermissions('Users', 'browseEnabled', { userId: 1 })).not.toThrow();
```